src/components/buttons/GoogleButton.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { 
+    Image,
+    Text, 
+    TouchableOpacity 
+} from 'react-native';
+import normalize from 'react-native-normalize';
+import { GOOGLE_LOGO } from '../../constants/icons';
+import { colors, fonts } from '../../theme';
+
+function GoogleButton({
```

Comment:
1. follow Es6 for functions
2. remove inline styles
3. use text from constants